src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -1368,6 +1373,16 @@ - (void)_setCachedAttributeValue:(MTRDeviceDataValueDictionary _Nullable)value f
 
     [clusterData storeValue:value forAttribute:path.attribute];
 
+    if (value != nil
+        && isFromSubscription
+        && !_receivingPrimingReport
```

Comment:
If https://github.com/project-chip/connectedhomeip/pull/33462/files merges first, we'll need change the `_receivingPrimingReport` check to a `HadSubscriptionEstablishedOnce(_internalDeviceState)` one.